src/AppBundle/PullRequests/BodyParser.php

```diff
@@ -105,7 +105,9 @@ public function isBackwardCompatible()
         $regex = sprintf(self::DEFAULT_PATTERN, 'BC breaks', '.+');
         $bcBreaks = $this->extractWithRegex($regex);
 
-        return 'yes' !== $bcBreaks;
+        return
+            'yes / no' !== $bcBreaks
```

Comment:
So, by default we consider a PR not backward compatible?

Reply:
I think this should be further improved in the future by using a checkbox instead